src/helpers/utils.js

```diff
@@ -461,3 +463,20 @@ export const getSourcesCollections = (sources, getPathById) =>
       return acc;
     }, {})).filter(source => source && source.children && source.children.length);
 
+// helper function to filter array and get unique values - arr.filter(distinct)
+export const distinct = (value, index, arr) => arr.indexOf(value) === index;
+
+export const renderTags = tagNames => (
+  tagNames?.length > 0 &&
+  <div className="unit-tags-bar">
+    {
+      tagNames
```

Comment:
[...new Set(tagNames)].map((tag, index) => .....

Reply:
we can take tags from unit and from Label